lib/proxy/util.rb

```diff
@@ -23,7 +23,7 @@ def start(&ensured_block)
       @task = Thread.new(@command, @input) do |cmd, input|
         status = nil
         Open3.popen3(*cmd) do |stdin, stdout, stderr, thr|
-          logger.info "[#{thr.pid}] Started task #{cmd}"
+          logger.info "[#{thr.pid}] Started task #{cmd.join(' ')}"
```

Comment:
This allows you to copy and paste commands from logs to simulate the command (when something goes wrong). Pretty handy.

Reply:
Sure.